src/api/submission/validations.py

```diff
@@ -853,7 +837,14 @@ def validate_observation_density(self):
 
     def validate_fish_count(self):
         obs = self.data.get("obs_belt_fishes") or []
-        num_fish = sum([obs.get("count") or 0 for obs in obs])
+        counts = []
+        for ob in obs:
+            try:
+                counts.append(int(ob.get("count") or 0))
```

Comment:
Shouldn't we use `cast_int` here?
Also: the previous comprehension approach would still work, wouldn't it? The idea here is to specifically catch invalid values?